veros/setup/acc_sector/acc_sector.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env python
+
+import veros
+import veros.tools
+
+
+class ACCSector(veros.Veros):
+    """A model using spherical coordinates with a partially closed domain representing the Atlantic and ACC.
```

Comment:
Could you please clarify in the docstring how this differs from the other ACC setup? What additional effects can you study with this?